homeassistant/components/slide_local/switch.py

```diff
@@ -0,0 +1,56 @@
+"""Support for Slide switch."""
+
+from __future__ import annotations
+
+from typing import Any
+
+from homeassistant.components.switch import SwitchDeviceClass, SwitchEntity
+from homeassistant.const import EntityCategory
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from . import SlideConfigEntry
+from .coordinator import SlideCoordinator
+from .entity import SlideEntity
+
+PARALLEL_UPDATES = 0
```

Comment:
the coordinator does not parallelize outgoing requests (e.g. swotch operations) can the api really deal with parallel requests